md_utils/frame/convert.py

```diff
@@ -6,9 +6,23 @@ def pdb_to_fasta(frame: Frame) -> None:
     writes one or more sequences from pdb file into a file in FASTA format
     :param path_to_pdb_file:
     :return: file in FASTA format
-    """
-    ...
+    """  
+    aa_code = {'CYS': 'C', 'ASP': 'D', 'SER': 'S', 'GLN': 'Q', 'LYS': 'K',
+     'ILE': 'I', 'PRO': 'P', 'THR': 'T', 'PHE': 'F', 'ASN': 'N', 
+     'GLY': 'G', 'HIS': 'H', 'LEU': 'L', 'ARG': 'R', 'TRP': 'W', 
+     'ALA': 'A', 'VAL':'V', 'GLU': 'E', 'TYR': 'Y', 'MET': 'M'}
+    
+    output = []
+    for i in range(len(frame.molecules)):
```

Comment:
frame.molecules return MoleculeSpan object. It's iterable object. 
You can just use: for molecule in frame.molecules.